### DISCUSSION TITLE
There should be separate options to allow GitHub Actions to (1) create pull requests (2) approve pull requests

### DISCUSSION THREAD

**Discussion by jrhemstad** [community/community#76707] [Actions]
### Select Topic Area

Product Feedback

### Body

In order to enable the `github-actions` bot to create PRs, you need to enable the setting: "Allow GitHub Actions to create and approve pull requests"

Anyone can create a PR on a public repo, so why would we also let the `github-actions` bot also _approve_ PRs just to let it create PRs? 

This seems like a clear violation of the Principle of Least Privilege.

There should be separate options for:
- Allow Github Actions to create pull requests
- Allow GitHub Actions to approve pull requests 

![image](https://github.com/community/community/assets/15221289/3883fde8-cf8f-4e4b-8ffd-c3efb77237b7)

**Comment by lasseheia** [comment]
Definitely want to see this sorted out as well!

I really think they should've kept the [original setting](https://github.blog/changelog/2022-01-14-github-actions-prevent-github-actions-from-approving-pull-requests/) and just added the [new one](https://github.blog/changelog/2022-05-03-github-actions-prevent-github-actions-from-creating-and-approving-pull-requests/), not replaced it. Additionally, when this feature is enabled at the organization level, it automatically applies to all new repositories. In my opinion, this feature should require explicit opt-in for each repository, irrespective of the organization's settings.
